legacy/src/main/java/org/opensearch/sql/legacy/query/planner/logical/node/TableScan.java

```diff
@@ -33,6 +37,9 @@ public PlanNode[] children() {
 
   @Override
   public <T> PhysicalOperator[] toPhysical(Map<LogicalOperator, PhysicalOperator<T>> optimalOps) {
+    if (LocalClusterState.state().getSettingValue(SQL_PAGINATION_API_SEARCH_AFTER)) {
```

Comment:
Maybe `SQL_PAGINATION_API` with "scroll" or "search_after"?

Reply:
Do you mean changing the setting key from `SQL_PAGINATION_API_SEARCH_AFTER` to `SQL_PAGINATION_API` and adjusting its value accordingly? This is a feature flag currently which is boolean and also used by serverless code to turn on and off the feature. We can address this in follow-up if not a major one.